optimize/backend/pom.xml

```diff
@@ -116,10 +133,12 @@
       <groupId>org.eclipse.jetty.ee10</groupId>
       <artifactId>jetty-ee10-servlet</artifactId>
     </dependency>
+    <!-- unused/declared
     <dependency>
       <groupId>org.eclipse.jetty.ee10</groupId>
       <artifactId>jetty-ee10-servlets</artifactId>
     </dependency>
+    -->
```

Comment:
I wonder what the difference between this and the one above is

Reply:
My understanding is that the one with the extra 's' extends the one without the extra 's' with more filters and functionalities. Link: https://javadoc.jetty.org/jetty-12/org/eclipse/jetty/ee10/servlets/package-summary.html